packages/plugins/src/plugins/eigenlayer/fetchers/deposit.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  ethereumNativeAddress,
+  ethereumNetwork,
+  NetworkId,
+  PortfolioAsset,
+  PortfolioElement,
+  PortfolioElementType,
+} from '@sonarwatch/portfolio-core';
+import { getEvmClient } from '../../../utils/clients';
+import { Cache } from '../../../Cache';
+
+import { eigenPodManagerAddress, platformId } from '../constants';
+import { getAddress } from 'viem';
+import { abi } from '../abi';
+import tokenPriceToAssetToken from '../../../utils/misc/tokenPriceToAssetToken';
+import BigNumber from 'bignumber.js';
+
+/**
+ * Returns the deposit positions for a given owner
+ * @param owner - The address of the owner
+ * @param cache - The cache instance
+ * @returns The deposit positions
+ */
+export const getDepositPositions = async (owner: string, cache: Cache) => {
+  const client = getEvmClient(NetworkId.ethereum);
+  const podManagerAddress = getAddress(eigenPodManagerAddress);
+
+  const podUserAddress = await client.readContract({
+    address: podManagerAddress,
+    abi: [abi.ownerToPod],
+    functionName: abi.ownerToPod.name,
+    args: [getAddress(owner)],
+  });
+
+  // If the user has no pod, the contract returns the ethereum native address address
```

Comment:
Is `pod` a concept I'm not aware of ? Or was this a simple typo ?

```suggestion
  // If the user has no pos, the contract returns the ethereum native address address
```

Reply:
Yep it's `pod`, this is the explication given by the AI agent of EigenLayer about what's a `pod` : 

> A pod, or EigenPod, is a smart contract deployed by a staker in the EigenLayer system. It's used to manage Ethereum staking and restaking. Key points about pods:
> - Each staker can deploy one pod.
> - Pods allow stakers to participate in EigenLayer's restaking mechanism.
> - They're controlled by their owners (also called "Pod Owners").
> - Pods handle validator operations, including deposits and withdrawals.
> - They interact with the EigenPodManager contract, which oversees all pods in the system.
> To create a pod, a staker calls the createPod() function in the EigenPodManager contract. This deploys a new EigenPod if the sender doesn't already have one.